python/foglamp/plugins/south/http_south/http_south.py

```diff
@@ -124,10 +155,10 @@ def plugin_shutdown(data):
         asyncio.ensure_future(app.shutdown())
         asyncio.ensure_future(handler.shutdown(60.0))
         asyncio.ensure_future(app.cleanup())
-        _LOGGER.info('South HTTP plugin shut down.')
     except Exception as e:
         _LOGGER.exception(str(e))
         raise
+    _LOGGER.info('South HTTP plugin shut down.')
```

Comment:
will this make a diff? it was last line in try block

Reply:
To maintain uniformity amongst all South plugin shutdown methods.